Added ability to use special characters for First Name and Last Name

SofiiaYevush opened the pull request:
Added the ability to use spaces " ", apostrophes " ' ", and hyphens " - "  in the User's First Name and Last Name fields. 
(Issue #2874)

Example of writing First and Last Name with a space, apostrophe, and hyphen in the fields:
![image](https://github.com/user-attachments/assets/5a304f7d-1317-4f42-aac1-d658059244d4)


User prompts have also been changed:
![image](https://github.com/user-attachments/assets/c7458b1e-daa9-4f72-8bfe-1e270b65ddbd)

SofiiaYevush committed:
Added ability to use special characters for First and Last name

Mav-Ivan commented:
implement issue to this Pr and move it to merge request
and remove label user story if this is an issue
![image](https://github.com/user-attachments/assets/131324db-7c22-478d-aab5-a624e790de14)

SofiiaYevush commented:
> implement issue to this Pr and move it to merge request and remove label user story if this is an issue ![image](https://private-user-images.githubusercontent.com/110425368/390747494-131324db-7c22-478d-aab5-a624e790de14.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI3OTIwNDQsIm5iZiI6MTczMjc5MTc0NCwicGF0aCI6Ii8xMTA0MjUzNjgvMzkwNzQ3NDk0LTEzMTMyNGRiLTdjMjItNDc4ZC1hYWI1LWE2MjRlNzkwZGUxNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEyOFQxMTAyMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03NDEwNTU4Y2IwNDE0ZGVkMWU1ZDcwMWE3MTdkOGM1ZmEzNTU5YzcwMjZmMTIyZDg4MDg0MTllOThjOTA5ZWY0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.xWUiXe6gjDcs2JuqOMxUooD665hgVWwoexuX2n2cOko)

Done. Implemented issue to this Pr, moved to 'merge request', removed label 'user story'.
